app/services/import_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module ImportHelpers
+  def find_location(country_iso)
+    Location.find_by!(iso: country_iso)
+  rescue StandardError
+    puts "Couldn't find Location with ISO: #{country_iso}"
+  end
+
+  def try_to_parse_date(date, expected_date_formats)
```

Comment:
maybe rename it to `find_date_with_format`?

Reply:
You can change it, for me both are good.

Reply:
I will leave it as it is.